src/sxastarter/src/assets/tailwind-layout.scss

```diff
@@ -1,3 +1,7 @@
+@import url('https://fonts.googleapis.com/css?family=Lora');
```

Comment:
I will ask Rob if we need to use all these fonts here

Reply:
Good